snow/networking/benchlist/benchlist_test.go

```diff
@@ -218,9 +218,6 @@ func TestBenchlistMaxStake(t *testing.T) {
 	require.Equal(b.benchedQueue.Len(), 2)
 	require.Equal(b.benchlistSet.Len(), 2)
 	require.Len(b.failureStreaks, 1)
-	fs := b.failureStreaks[vdrID2]
-	fs.consecutive = threshold
-	fs.firstFailure = now
```

Comment:
why dropping this?
Why not
```
b.failureStreaks[vdrID2].consecutive = threshold
b.failureStreaks[vdrID2].firstFailure = now
```

Reply:
That's not valid golang- `b.failureStreaks[vdrID2]` is not addressable

Reply:
Right. So how about
```
	b.failureStreaks[vdrID2] = failureStreak{
		consecutive:  threshold,
		firstFailure: now,
	}
```

What I am saying is that maybe test creator intent was to set the failureStreak specific values.